### DISCUSSION TITLE
Toggle a value/variable?

### DISCUSSION THREAD

**Discussion by ivanh66** [nguyenquyhy/Flight-Tracker-StreamDeck#325] [General]
Is there anyway for the toggle parameter to alternate values on a second button push?

For example, I am trying to toggle the following: 

COM2_RECEIVE_SELECT

The toggle parameter for above is 0=receive off and 1= receive on. 

Is there a certain syntax so that if you keep hitting the button it alternates between sending a 0 and 1?

**Comment by DonQuilmi** [comment]
It's not presently possible. There is a workaround using the Stream Deck "multi-action commuter" (translating from French, so I'm not sure it's exactly called that).
1st press triggers an action, 2nd another one. The downside is the ON/OFF icon will only react to the stream decks actions, and won't update if you use the simulator command (or a dedicated keyboard key/joystick button).
